satpy/etc/composites/msu_gsa.yaml

```diff
@@ -0,0 +1,36 @@
+sensor_name: visir/msu-gsa
+
+composites:
+  overview_raw:
+    compositor: !!python/name:satpy.composites.RGBCompositor
+    prerequisites:
+    - name: C03
+    - name: C03
```

Comment:
The standard `overview` composite is using `0.6, 0.8, 10.8` um channels, so the first channel should be `C01`. Hard to say which it should be for the green channel.